src/common/settings.js

```diff
@@ -26,6 +26,7 @@ var loadDefault = function(version) {
         showTrayIcon: false,
         trayIconTheme: '',
         disablewebsecurity: true,
+        autostart: false,
```

Comment:
node-auto-launch uses registry entry. So I think that the app should not keep the config by itself. How about read the state by node-auto-launch when showing the settings page?